libraries/cli/include/cli/config_jsons/devnet/devnet_config.json

```diff
@@ -11,19 +11,23 @@
       "http_port": 9777,
       "ws_port": 6777
     },
+    "ddos_protection": {
+      "vote_accepting_periods": 5,
+      "vote_accepting_rounds": 5,
+      "vote_accepting_steps": 0,
+      "log_packets_stats": true,
+      "packets_stats_time_period_ms": 60000,
+      "peer_max_packets_processing_time_us": 0
```

Comment:
This value needs to be configured - I am still monitoring devnet to decide what number to use